src/OlStyleParser.ts

```diff
@@ -727,13 +727,13 @@ export class OlStyleParser implements StyleParser<OlStyleLike> {
         let minScale = rule?.scaleDenominator?.min;
         let maxScale = rule?.scaleDenominator?.max;
         let isWithinScale = true;
-        if (minScale !== undefined || maxScale !== undefined) {
+        if (minScale || maxScale) {
```

Comment:
Can we safely ignore `0`? Which is false in JS.

Reply:
If your min scale limit is 0 features should always be rendered. Having a max scale limit of 0 then this technically will be handled 'wrong', because semantically this would mean that features are never rendered, but I'd argue that this is not a proper use case.